packages/shared/src/types/settings.ts

```diff
@@ -31,4 +33,7 @@ export interface SettingsState extends SettingsProps {
 
   // Welcome Screen
   onWelcomeScreenEntered: (shown: boolean) => void;
+
+  // Destination Chain
+  onDestinationChainSelected: (chainId: number) => void;
```

Comment:
@chybisov This was a single state use case, should I make a different store called `useMultisigStore` or since this is a minor state this would work?

Reply:
got it. I'll move it